db/video/cfr/README.txt

```diff
@@ -5,4 +5,6 @@ Get frames from video:
 ffmpeg -i test.mp4 %03d.png
 
 Build video from frames:
-ffmpeg -r 60 -i %03d.png -c:v libx264 -g 10 -keyint_min 10 -vf fps=60 -pix_fmt yuv420p test.mp4
\ No newline at end of file
+ffmpeg -r 60 -i %03d.png -c:v libx264 -g 10 -keyint_min 10 -vf fps=60 -pix_fmt yuv420p test.mp4
+
+Avi files are transcoded mp4 files.
```

Comment:
Can you add a command used to transcode them?

Reply:
Done